settings.sample.py

```diff
@@ -98,10 +98,11 @@ def env(key, default, cast=str):
 
 # Uncomment this option to enable adding an issue comment
 # whenever a pull request passes all checks.
-# OK_COMMENT = env('LINTREVIEW_OK_COMMENT',
-#                 ':+1: No lint errors found.')
+# eg: OK_COMMENT = ':+1: No lint errors found.'
+OK_COMMENT = env('LINTREVIEW_OK_COMMENT', '')
 
 # Enable to apply a label when updating build status.
 # Pull requests that fail will have the label removed.
 # Customize the label name when label statuses are enabled.
-# OK_LABEL = env('LINTREVIEW_OK_LABEL', 'No lint errors')
+# eg: OK_LABEL = 'No lint errors'
+OK_LABEL = env('LINTREVIEW_OK_LABEL', '')
```

Comment:
This one too. Having these default to the same values in the code, would be nice, as there are fewer cases to handle later on.